app/models/restaurant.rb

```diff
@@ -18,7 +20,9 @@ def self.top_restaurants
     top_array = []
     restaurants = Restaurant.all
     restaurants.each do |restaurant|
-      top_hash[restaurant] = restaurant.reviews.sum(:rating)/restaurant.reviews.count
+      if restaurant.reviews.count > 0
```

Comment:
this is redundant flow control to what is already in the instance method and is therefore unnecessary